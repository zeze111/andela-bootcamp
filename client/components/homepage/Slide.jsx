import React from 'react';
import { Slider, Slide } from 'react-materialize';
import slide1 from '../../assets/images/recipe.png';
import slide2 from '../../assets/images/food.jpg';
import slide3 from '../../assets/images/food2.jpg';
import slide4 from '../../assets/images/des.jpg';

export default () => {
  return (
    <div className="container">
      <div className="row" style={{marginTop: '0', paddingTop: '1em'}}>
        <Slider>
          <Slide
            src={slide1} className="responive-img" 
            placement= "left" >
            <p className="blue-text" style={{marginTop: 215 + 'px', fontSize: 30 + 'px' }}> Sign Up For Free! </p>
          </Slide>
          <Slide 
            src={slide2} className="responive-img"
            placement = "left">
            <p className="light grey-text text-lighten-3" style={{ fontSize: 30 + 'px' }}> Come indulge your taste buds </p>
          </Slide>
          <Slide 
            src={slide3} className="responive-img" 
             placement = "center">
            <p className=" light grey-text text-lighten-3" style={{ fontSize: 30 + 'px' }}> Share your culinary talent </p>
          </Slide>
          <Slide 
            src={slide4} className="responive-img" 
            placement = "right">
            <p className="light blue-text" style={{ fontSize: 30 + 'px' }}> Yummy treats for your meets </p>
          </Slide>
        </Slider>
      </div>
    </div>
  );
}
