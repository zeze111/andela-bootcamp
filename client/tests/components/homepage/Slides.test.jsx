import React from 'react';

import Slide from '../../../components/homepage/Slide';
import store from '../../../store';

describe('Test for Slide', () => {
  it('should render component', () => {
    const wrapper = shallow(<Slide /> );
    expect(wrapper).toMatchSnapshot()
  });
});
