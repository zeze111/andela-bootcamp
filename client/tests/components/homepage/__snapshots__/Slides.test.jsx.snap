// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Slide should render component 1`] = `
<div
  className="container"
>
  <div
    className="row no-top one-top"
  >
    <Slider
      fullscreen={false}
      indicators={true}
      interval={6000}
    >
      <Slide
        className="responive-img"
        placement="left"
        src="test-file-stub"
      >
        <p
          className="red-text slide-pad slide-text"
        >
           Sign Up For Free!
        </p>
      </Slide>
      <Slide
        className="responive-img"
        placement="left"
        src="test-file-stub"
      >
        <p
          className="light grey-text text-lighten-3 slide-text"
        >
           Come indulge your taste buds
        </p>
      </Slide>
      <Slide
        className="responive-img"
        placement="center"
        src="test-file-stub"
      >
        <p
          className="light grey-text text-lighten-3 slide-text"
        >
           Share your culinary talent
        </p>
      </Slide>
      <Slide
        className="responive-img"
        placement="right"
        src="test-file-stub"
      >
        <p
          className="light grey-text text-lighten-5 slide-text"
        >
           Yummy treats for your meets
        </p>
      </Slide>
    </Slider>
  </div>
</div>
`;
